Lib/test/test_interpreters/test_api.py

```diff
@@ -54,6 +54,10 @@ def test_in_main(self):
         self.assertIsInstance(interp, interpreters.Interpreter)
         self.assertIn(interp, interpreters.list_all())
 
+        # GH-126221: Passing an invalid unicode character used to cause a SystemError
+        with self.assertRaises(UnicodeEncodeError):
+            _interpreters.create('\udc80')
```

Comment:
```suggestion
		self.assertRaises(UnicodeEncodeError, _interpreters.create, '\udc80')
```

We could reduce the diff :)

Reply:
HA, I hate when GH commit tabs instead of spaces :D @ZeroIntensity. You should fix them (I don't know why it doesn't convert my tabs to spaces...)